src/lib/FreeC/Backend/Agda/Converter/FuncDecl.hs

```diff
@@ -31,17 +35,22 @@ convertFuncDecl decl = localEnv $ sequence [convertSignature decl]
 -- | Converts the type signature of the given function to an Agda type
 --   declaration.
 convertSignature :: IR.FuncDecl -> Converter Agda.Declaration
-convertSignature (IR.FuncDecl _ (IR.DeclIdent srcSpan name) tVars args retType _)
-  = Agda.funcSig <$> ident <*> convertFunc tVars types retType
- where
-  types = map IR.varPatType args
-  ident = lookupUnQualAgdaIdentOrFail srcSpan IR.ValueScope name
+convertSignature (IR.FuncDecl _ (IR.DeclIdent s name) tVars args rType _) = do
+  let types = map IR.varPatType args
+  let ident = lookupUnQualAgdaIdentOrFail s IR.ValueScope name
+  decArg <- inEnv $ lookupDecArgIndex name
```

Comment:
Yes, my plan was to use `lookupDecArg` only temporarily to test the conversion code. Since i have to use the analyis modules anyway I will change this module to use `identifyDecArgs`. 

I shortened the names in this function, because brittany did not enforce the 80 column limit. The code after formatting with the long names is (line 2 is 87 characters long!):
```haskell
convertSignature :: IR.FuncDecl -> Converter Agda.Declaration
convertSignature (IR.FuncDecl _ (IR.DeclIdent srcSpan name) typeVars args returnType _)
  = do
    let types = map IR.varPatType args
    let ident = lookupUnQualAgdaIdentOrFail srcSpan IR.ValueScope name
    decArg <- inEnv $ lookupDecArgIndex name
    Agda.funcSig <$> ident <*> convertFunc decArg typeVars types returnType
```

Reply:
That Brittany does not break long left-hand sides annoys me as well. I would still go for the long names, though. Meaningful variable names improve the comprehensibility of code more than long lines hurt readability. One way to avoid long lines is to use selector functions of the `IR`. I've now [added some more notes](https://github.com/FreeProving/guidelines/commit/2aeb86fe57557ebb07f24fb61ac63db16c49c7bf) about the naming conventions to the Haskell Styleguide and clarified that it is okay if left-hand sides exceed the line length limit.